### DISCUSSION TITLE
Intel NCS

### DISCUSSION THREAD

**Discussion by kwh703** [dcmartin/motion-ai#18] [Q&A]
Can you provide some more info on how to get motion AI working with the Intel NCS2 on RPi4? I wasn’t clear based on what’s in the documentation how to add the movidius/NCS support.

**Comment by dcmartin** [comment]
Sorry for the delay, but I have not progressed the NCS2 integration at this time; it is not packaged and not simple to configure; not the least of which are the Intel instructions for utilization with OpenCV; TBH, I don't have time to pursue the Intel NCS2.
